core/templates/pages/community-dashboard-page/community-dashboard-page.scripts.ts

```diff
@@ -18,6 +18,8 @@
 
 require('pages/community-dashboard-page/community-dashboard-page.module.ts');
 require('App.ts');
+require('base-components/oppia-root.directive.ts');
+require('base-components/base-content.directive.ts');
```

Comment:
I see `oppia-root.directive` should be added and that is added in each script but why the addition of `base-content.directive`?

Reply:
I ran into an error in some-pages that were a result of base-content not being defined. This solved it. Base content is used in almost all modules, I think it is better to have it here.

Reply:
What change we did that resulted in this error? This error wasn't there before.

Reply:
This is so weird. I am not able to reproduce that error. The last time I tried I was not able to go past the signup screen. It would keep refreshing.

Reply:
@bansalnitish I have removed those extra additions of `base-directive` as I was not able to reproduce the error.

Reply:
Cool, thanks :D